docs/external-references.md

```diff
@@ -18,27 +18,78 @@ MyST supports referencing rich content in a growing number of formats, including
 [](./cross-references.md) for referencing content in your project and [](./citations.md) to cite scholarly work and create bibliographies.
 ```
 
+(myst-xref)=
 ## Referencing external MyST projects
 
 When using the HTML renderer for MyST, an API is provided for the deployed site.
-This provides pre-parsed, structured content as an AST that can be included and rendered in a tooltip.
+This provides pre-parsed, structured content as an AST that can be included in other projects and rendered in a tooltip.
 
 ```{tip}
 Try adding `.json` at the end of the URL on this page. The data is structured and provides authors, license information,
 as well as the full content in a parsed form that can be used for an inline reference on external pages.
 ```
 
```

Comment:
````suggestion
```{tip} Add `.json` to the end of a MyST URL to see its data 
All MyST pages come with a structured data representation that provides authors, license information,
as well as the full content in a parsed form that can be used for an inline reference on external pages.
Try adding `.json` at the end of the URL on this page.
```

````